project.py

```diff
@@ -75,6 +76,25 @@ def email_domain_and_user_count(string_of_emails):
     
     return dict(sorted(dict_of_emails_and_users_count.items()))
 
+    
+def find_duplicate_emails(data):
+    data= './data.csv'

```

Comment:
This is an issue because it over-writes the value of data passed in the parameter